string strVal

int64 num

int64 a
int64 b
int64 c